docs/src/main/paradox/mongodb.md

```diff
@@ -38,7 +38,8 @@ The table below shows direct dependencies of this module and the second tab show
 
 In the code examples below we will be using Mongo's support for automatic codec derivation for POJOs.
 For Scala we will be using a case class and a macro based codec derivation.
-For Java a codec for POJO is derived using reflection.
+For Java a codec for POJO is derived using reflection. We assume that you already have an 
```

Comment:
Due to us no longer manually doing `val actorSystem = ActorSystem.create()` in test code since we get `System` from `TestKit` I have updated the doc's here to mention you need an `ActorSystem`.